src/components/LoginForm.jsx

```diff
@@ -19,9 +19,9 @@ class LoginForm extends Component {
     } = this.props;
     return (
       <div className='login-form'>
-        <form>
-          <InputTextField fieldName='Email' placeholder='Email' value={email} onChange={loginFormUpdate}/>
-          <InputTextField type='password' placeholder='Password' fieldName='Password' value={password} onChange={loginFormUpdate}/>
+        <form className='login-form__form'>
```

Comment:
not everything has to have __. this could just be login-form

Reply:
the div it is enclosed in also has the class login-form...?

Reply:
how about __container then?